pinot-common/src/main/java/org/apache/pinot/common/config/provider/TableCache.java

```diff
@@ -71,6 +73,8 @@ public class TableCache implements PinotConfigProvider {
   private static final String LOWER_CASE_OFFLINE_TABLE_SUFFIX = OFFLINE_TABLE_SUFFIX.toLowerCase();
   private static final String LOWER_CASE_REALTIME_TABLE_SUFFIX = REALTIME_TABLE_SUFFIX.toLowerCase();
 
+  private static final String DEFAULT_DATABASE_PREFIX = CommonConstants.DEFAULT_DATABASE + ".";
```

Comment:
Are the changes in this class required?

Reply:
They are required to handle the `actualTableNameWithoutDot` and `default.actualTableNameWithoutDot` table name equivalence.